### DISCUSSION TITLE
How do I get the actual shipping label cost when purchased through Etsy

### DISCUSSION THREAD

**Discussion by casualbobuk** [etsy/open-api#811] [Q&A]
Hi, I can retrieve the amount the customer paid for shipping just fine, but I can't seem to find how to retrieve what I actually paid for a shipping label when purchased through Etsy. Is this possible please?

Thank you

**Comment by juneyeeleow** [comment]
I need this too.
Tried with V2 like 1-2 years ago back then it wasn't possible to retrieve the orders with fee, tax, and adjusted total and label cost.
Now it's possible to retrieve orders, fee and tax but not label cost...

Please someone make it happen.
